ee/docker-compose.ch.arm64.yml

```diff
@@ -0,0 +1,98 @@
+version: '3'
+
+services:
+    db:
+        image: postgres:12-alpine
+        environment:
+            POSTGRES_USER: posthog
+            POSTGRES_DB: posthog
+            POSTGRES_PASSWORD: posthog
+        ports:
+            - '5432:5432'
+    redis:
+        image: 'redis:alpine'
+        ports:
+            - '6379:6379'
+    clickhouse:
+        # Build with: yarn arm64:build:clickhouse
+        image: clickhouse-dev-arm64:latest
```

Comment:
Is there a simple way we can avoid having another docker compose file? Ie choosing an image based on arch env var, or using a multi arch image?